assembl/templates/views/messageSend.tmpl

```diff
@@ -1,9 +1,10 @@
 <% if (canPost) { %>
-    <% if ( !reply_message_id ) { %>
```

Comment:
I believe it should work _because_ the same view is used, but yes we need to check.